app/javascript/chat/chat.jsx

```diff
@@ -1341,6 +1329,45 @@ export default class Chat extends Component {
                 ) : null}
               </span>
             </button>
+            {this.state.isTagModerator ? (
+              <button
+                className="chat__channelssearchtoggle "
+                aria-label="Toggle request manager"
+                onClick={this.toggleModalCreateChannel}
+                data-content="sidecar-joining-request-manager"
+              >
+                <span
```

Comment:
refactor (blocking): This span acting as a button appears to be redundant given that it's contained within a `<button />`.